src/main/java/frc/robot/subsystems/LowScoringSubsystem.java

```diff
@@ -45,10 +46,10 @@ public LowScoringSubsystem() {
     conveyorMotor.setIdleMode(IdleMode.kBrake);
     rollerMotor.setInverted(true);
 
-    rollerEncoder = rollerMotor.getEncoder();
+    rollerEncoder = rollerMotor.getEncoder(EncoderType.kNoSensor, 0);
```

Comment:
Maybe add a comment here that explains since we're not using a NEO nor an external encoder here we have to do this so the SparkMAX doesn't throw sensor errors.